### PR TITLE
Merkle Rewards System: Add v8

### PR DESCRIPTION
**Pull request by jshufro**

### CODE REVIEW

File: Merkle Rewards System/spec-history.md [hunk 1]
```diff
@@ -24,4 +25,12 @@ This table shows which rewards specification was used for each rewards interval
 | 49-59 | [v3](./legacy/rewards-calculation-spec-v3.md) |
 | 60-75 | [v4](./legacy/rewards-calculation-spec-v4.md) |
 | 76-117 | [v5](./legacy/rewards-calculation-spec-v5.md) |
-| 118+ | [v6](./rewards-calculation-spec.md) |
+| 118-? | [v6](./legacy/rewards-calculation-spec-v6.md) |
+| ?-118 | [v7](./legacy/rewards-calculation-spec-v7.md) |
```
Comment (jshufro) [inline Merkle Rewards System/spec-history.md:29]: not sure if there's an easy way to find the right numbers for prater. might have to check the rewards files, which i don't have readily available.

File: Merkle Rewards System/spec-history.md [hunk 2]
```diff
@@ -12,8 +12,9 @@ This table shows which rewards specification was used for each rewards interval
 | 5 | [v3](./legacy/rewards-calculation-spec-v3.md) |
 | 6-7 | [v4](./legacy/rewards-calculation-spec-v4.md) |
 | 8-11 | [v5](./legacy/rewards-calculation-spec-v5.md) |
-| 12+ | [v6](./rewards-calculation-spec.md) |
-
+| 12-? | [v6](./legacy/rewards-calculation-spec-v6.md) |
+| ?-17 | [v7](./legacy/rewards-calculation-spec-v7.md) |
```
Comment (jshufro) [inline Merkle Rewards System/spec-history.md]: anyone know off hand which interval first used v7?

Comment (0xfornax) [reply Merkle Rewards System/spec-history.md]: 15

File: Merkle Rewards System/spec-history.md (hunk 1 above)
Comment (0xfornax) [reply Merkle Rewards System/spec-history.md:29]: I also don't have the prater rewards files but I think it's ok to ignore that as it's deprecated.

**Pull request merged**